### PR TITLE
Add ARM64 SIMD extmul and extadd operations

### PR DESCRIPTION
**Pull request by gergocs**
Signed-off-by: Gergo Csizi gergocs@inf.u-szeged.hu

### CODE REVIEW

File: src/jit/SimdArm64Inl.h [hunk 1]
```diff
@@ -413,6 +445,26 @@ static void emitBinarySIMD(sljit_compiler* compiler, Instruction* instr)
     case ByteCode::I16X8GeUOpcode:
         simdEmitOp(compiler, SimdOp::ucmge | SimdOp::H8, dst, args[0].arg, args[1].arg);
         break;
+    case ByteCode::I16X8ExtmulLowI8X16SOpcode:
+        simdEmitOp(compiler, SimdOp::sxtl | (0x1 << 19), args[0].arg, args[0].arg, 0);
```
Comment (zherczeg) [inline src/jit/SimdArm64Inl.h:449]: Is this change args[0]?

Comment (gergocs) [reply src/jit/SimdArm64Inl.h:449]: yes

Comment (zherczeg) [reply src/jit/SimdArm64Inl.h:518]: It could be a problem when register allocation is implemented. It should use dst and maybe a tmp register instead.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**